apps/web/src/pages/ItTrainingFundPage/ItTrainingFundPage.tsx

```diff
@@ -472,7 +469,7 @@ export const Component = () => {
                           data-h2-font-weight="base(bold)"
                           color="secondary"
                           external
-                          href={signUpUrl[locale]}
+                          href={paths.instructorLedTraining()}
                         >
```

Comment:
I believe the link text will need to be updated to this as well, to match the new href

![image](https://github.com/user-attachments/assets/b8c29ece-6aad-4251-9a5d-1941fcd080ed)

Reply:
Nice catch, changed the text here [49f7e92](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12083/commits/49f7e92309442c9e1cb45e58192b69083004d6c9)